sonic-thermalctld/setup.py

```diff
@@ -23,7 +23,8 @@
     tests_require=[
         'mock>=2.0.0; python_version < "3.3"',
         'pytest',
-        'pytest-cov'
+        'pytest-cov',
+        'sonic-platform-common'
```

Comment:
@jleveque  this dependency is causing https://github.com/Azure/sonic-buildimage/pull/7143 sonic-buildimage to fail. 
Do we need to remove it or add something ?

Reply:
PR to add dependency in sonic-buildimage [here](https://github.com/Azure/sonic-buildimage/pull/7181)